elb.tf

```diff
@@ -35,11 +45,22 @@ resource "aws_security_group_rule" "elb_allow_egress" {
 resource "aws_lb" "this" {
   name_prefix                      = substr(var.name, 0, 6)
   enable_cross_zone_load_balancing = true
-  internal                         = false
+  internal                         = false #tfsec:ignore:AWS005
   load_balancer_type               = "application"
   security_groups                  = [aws_security_group.elb.id]
   subnets                          = var.public_subnets
   tags                             = var.tags
+
+  dynamic "access_logs" {
```

Comment:
are you sure you can do this? The docs say "An Access Logs block", which I interpret as singular. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb#access_logs

Reply:
yep, we use this pattern a lot. Pay attention to how I build the local it iterates on.